extension/index.js

```diff
@@ -38,10 +42,19 @@ async function init() {
     addDependencies(createBox('Dev Dependencies'), devDependencies);
   }
 
-  if (dependencies.length && !pkg.private) {
-    const link = html`<a class="npmhub-anvaka btn btn-sm">Visualize full tree`;
-    link.href = `http://npm.anvaka.com/#/view/2d/${esc(pkg.name)}`;
-    dependenciesBox.insertBefore(link, dependenciesBox.firstChild);
+  if (!pkg.private) {
+    const realPkg = await window.backgroundFetch(getPkgUrl(pkg.name));
+    if (realPkg.name) { // if 404, realPkg === {}
```

Comment:
What if the response is something other than 200 or 404? Is `realPkg` guaranteed to be an object?

If not, maybe `if (realPkg && realPkg.name)` would be safer.